articles/azure-functions/durable-functions-http-api.md

```diff
@@ -193,6 +193,9 @@ The **HTTP 202** response also includes a **Location** response header that refe
 
 You can also query all instances status. Remove the `instanceId` from the 'Get instance status' request. The parameters are the same as the 'Get instance status.' 
 
+One thing to remember is taht connection and code are optional. If you have anonymous auth on the function then code isn't required.
```

Comment:
Typo: taht --> that

Also, I suggest putting **code** and **connection** in back ticks so that they are formatted like `code` and `connection`.